uma/uma.py

```diff
@@ -630,7 +653,10 @@ def create_pay_req_response(
     )
     payer_identifier = request.payer_data["identifier"] if request.payer_data else None
     if not payer_identifier and request.is_uma_request():
-        raise InvalidRequestException("Missing payer identifier in request")
+        raise InvalidRequestException(
+            "Missing payer identifier in request",
```

Comment:
Non-blocking, but this is one where I could see adding extra json data for which fields are missing that were expected.